packages/uikit-react-native-foundation/src/ui/GroupChannelMessage/Message.file.video.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import type { SendbirdFileMessage } from '@sendbird/uikit-utils';
+import { getAvailableUriFromFileMessage } from '@sendbird/uikit-utils';
+
+import Box from '../../components/Box';
+import Icon from '../../components/Icon';
+import ImageWithPlaceholder from '../../components/ImageWithPlaceholder';
+import PressBox from '../../components/PressBox';
+import createStyleSheet from '../../styles/createStyleSheet';
+import useUIKitTheme from '../../theme/useUIKitTheme';
+import MessageContainer from './MessageContainer';
+import type { GroupChannelMessageProps } from './index';
+
+type Props = GroupChannelMessageProps<
+  SendbirdFileMessage,
+  {
+    fetchThumbnailFromVideoSource: (uri: string) => Promise<{ path: string } | null>;
+  }
+>;
+
+const VideoFileMessage = (props: Props) => {
+  const { onPress, onLongPress, variant = 'incoming' } = props;
+
+  const { colors, palette } = useUIKitTheme();
+  const uri = getAvailableUriFromFileMessage(props.message);
+  const { thumbnail, loading } = useRetry(() => props.fetchThumbnailFromVideoSource(uri));
+
+  return (
+    <MessageContainer {...props}>
+      <Box style={styles.container} backgroundColor={colors.ui.groupChannelMessage[variant].enabled.background}>
+        <PressBox activeOpacity={0.8} onPress={onPress} onLongPress={onLongPress}>
+          {loading ? (
+            <Box backgroundColor={colors.onBackground04} style={styles.image} />
+          ) : (
+            <ImageWithPlaceholder source={{ uri: thumbnail ?? 'invalid-image' }} style={styles.image} />
+          )}
+
+          {(loading || thumbnail !== null) && (
+            <Box style={styles.iconContainer} alignItems={'center'} justifyContent={'center'}>
+              <Icon
+                icon={'play'}
+                size={28}
+                color={palette.onBackgroundLight02}
+                containerStyle={[styles.playIcon, { backgroundColor: palette.onBackgroundDark01 }]}
+              />
+            </Box>
```

Comment:
Absolute container should be inside of sized box

Reply:
resolved in https://github.com/sendbird/sendbird-uikit-react-native/pull/93/commits/ce8a81f80d0a8d8c70f92c5ba4a7052689255a34